src/components/TopicLoader/TopicLoader.js

```diff
@@ -22,7 +22,7 @@ const propTypes = {
     push: PropTypes.func,
     replace: PropTypes.func,
   }),
-  apiKey: PropTypes.string.isRequired,
+  apiKey: PropTypes.string,
   apiKeyName: PropTypes.string.isRequired,
```

Comment:
apiKeyName is also not required and default value is 'key'

Reply:
Indeed I adapt it